docs/concepts/sensor-models/sensor-models.md

```diff
@@ -0,0 +1,90 @@
+
+- Image Reference System: a 2-dimensional (column/row or line/sample) reference system with a defined (0,0) origin, typically at the upper-left of the the frame
+- Distorted Focal Plane Reference System: a 2-dimensional (x,y) reference system nominally represented in Cartesian space and measured millimeters.  This reference system is used to account for the distortion in an image reference system.
+- Sensor Reference System: a 3-dimensional (x,y,z) reference system in which the z axis typically follows the sensor's boresight (if it is an optical sensor) as well as x and y axes that are traditionally parallel to the sensor's x and y axes.
+- Spacecraft Reference System: a 3-dimensional (x,y,z) reference system in which the origin is centered on the spacecraft's center of mass. Axes are often represented using 𝜔, 𝜙, 𝜅, which are analogous to yaw, pitch, and roll, but instead of representing axial rotations with respect to onboard navigation, they are expressed with respect to axes of an arbitrary coordinate system.
+- Body Centered Body Fixed (BCBF): a 3-dimensional (x,y,z) reference system in which the origin is centered on the target's center of mass.
+
+It is important to note that a reference system does not provide a description of an object's location, but it instead provides a _means_ for describing locations.  When locations are described using a reference system, the result is a "coordinate reference frame," which offers information related to an object's position, orientation, and velocity at a single instant in time.
+
+!!! INFO "Reference System vs Reference Frame"
+    A __reference system__ provides an origin, axes / planes, and a fixed point that allows for the description of an object.  A __reference frame__ is a description within the context of that reference system, and provides the location of an object at a given moment in time.
+
+Finally, reference frames often have dependencies on other reference frames, which results in a "frame chain" that must be calculated. For example, consider a common scenario in which a camera is intended to record an image of a planetary body.  The camera is mounted on a gimbal, which is mounted on the spacecraft.  In order to move from the image plane to a BCBF system, it is necessary to know the location of the camera and the planetary body.  However, in order to know the location of the camera, it is necessary to know the extrinsics not only of the camera, but also of the gimbal and the spacecraft to which it is mounted.  The dependency chain that is formed is known as a "frame chain," and the entire chain of extrinsics must be calculated in order to accurately model the objects' relative locations.
```

Comment:
I would take this into a new section and then provide a simple example. For example, take an imaginary sensor, fixed to a spacecraft, and describe the chain of transformations to get from the sensor position in the bodies reference frame. Just include frame chain stuff, not distortion, focal/pixel, etc. Just transform the position of the sensor into the body frame.